lib/modules/datasource/python-version/readme.md

```diff
@@ -0,0 +1 @@
+This datasource returns Python releases from the [python.org](https://www.python.org/api/v2/downloads/release/) API.
```

Comment:
```suggestion
This datasource returns Python releases from the [python.org API](https://www.python.org/api/v2/downloads/release/).
```

Let's put the API text into the link, so it's clear where you'll end up after clicking the link.